### ISSUE TITLE
allowCompoundWords does not take into consideration user/workspace defined words

### ISSUE THREAD

**Issue by popadi** [streetsidesoftware/vscode-spell-checker#2570]
With the following config:
```
{
    "cSpell.allowCompoundWords": true,
    "cSpell.ignorePaths": [
        "node_modules",
        "**/node_modules",
        "**/node_modules/**",
        "node_modules/**",
        "vscode-extension",
        ".git",
    ],
    "cSpell.words": [
        "fcm",
    ]
}
```

I would expect for the below snippet to not be flagged:
```
def process_fcmevents():
    ...
```

However, it is flagged: `"fcmevents": Unknown word.`
It only works if I add `fcmevents` in `words`.
Is this expected?

**Comment by Jason3S** [comment]
@popadi,

`allowCompoundWords` only compounds words within the same dictionary.

The `words` dictionary and the English dictionary are not considered the same dictionary.
